tests/integration/test_workflow.py

```diff
@@ -171,7 +171,7 @@ def assert_accuracy(self, accuracy):
 class NnrfWorkflowTestCase(WorkflowTestCase):
     def setUp(self):
         super().setUp()
-        self.min_accuracy = 0.58
+        self.min_accuracy = 0.46
```

Comment:
As discussed, revert these changes causing accuracy drop because the time saved is only ~2 mins in all tests combined.